Java/playwright/pom.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.example</groupId>
+  <artifactId>playwright</artifactId>
+  <version>1.0-SNAPSHOT</version>
```

Comment:
is this specifying the version of our playwright java integration? Shouldn't this be 4.4?

Reply:
Updated the version. Its should be 4.2.2, I guess that's the one being used in prod currently

Reply:
Sorry missed this. versions denoted as `x-SNAPSHOT` are QA builds. We should use the latest release on Agora should be `4.4.0`